### ISSUE TITLE
Plugins page: Should we bring back blocklist?

### ISSUE THREAD

**Issue by phated** [gulpjs/gulpjs.github.io#114]
In the initial implementation of the new plugin page, I didn't include the gulpplugin blocklist. 

Should we bring it back?

**Comment by donghwipark** [comment]
@phated Does blocklist related to the deprecated plugins?

**Comment by phated** [comment]
@donghwipark no, the blocklist was a list maintained by the gulp core team of plugins that don't follow the guidelines or should not exist or be used. We never updated the guidelines and the team stopped watching for bad plugins. Also, some plugin authors yelled at us.

**Comment by donghwipark** [comment]
@phated I understood what the blocklist means and the situation!!
